cloud/scaling.rst

```diff
@@ -0,0 +1,58 @@
+Scaling
+#######
+
+Citus Cloud provides self-service scaling to deal with increased load. The web interface makes it easy to either add new worker nodes or increase existing nodes' memory capacity.
+
+For most cases either approach to scaling is fine and will improve performance. However there are times you may want to choose one over the other. If the cluster is reaching its disk limit then adding more nodes is the best choice. Alternately, if there is still a lot of headroom on disk but performance is suffering, then scaling node RAM is the way to go.
+
+Both adjustments are available in the formation configuration panel of the settings tab:
+
+.. image:: ../images/cloud-formation-configuration.png
+
+Clicking the node change link provides two sliders:
+
+.. image:: ../images/cloud-nodes-slider.png
+
+The first slider, **count**, scales out the cluster by adding new nodes. The second, **RAM**, scales it up by changing the amount of RAM on existing nodes.
+
+For example, just drag the slider for node count:
```

Comment:
I would have liked to see these different sliders in the two different sections, but I think it looks good for now.